packages/forms-web-app/src/dynamic-forms/controller.test.js

```diff
@@ -522,7 +522,7 @@ describe('dynamic-form/controller', () => {
 			getJourney.mockReturnValue(mockCompletedJourney);
 
 			await submit(req, res);
-			expect(res.render).toHaveBeenCalledWith('./dynamic-components/submission-screen/index');
+			expect(res.redirect).toHaveBeenCalledTimes(1);
```

Comment:
Did you decide not to do a `toHaveBeenCalledWith` because the referenceId changes each time? If so you can check the string matches a regex with something like
```
expect(res.redirect).toHaveBeenCalledWith(expect.stringMatching(/^\/manage-appeals\/.+\/questionnaire-submitted\/$/))
```
If not feel free to ignore :)